### PR TITLE
Permettre le test de la non-calculabilité

### LINKED ISSUE #22: Prendre en compte la non-calculabilité
Un droit non calculable a une valeur de type chaîne de caractères, qui est un identifiant de la raison pour laquelle il n'a pas pu être calculé. Ludwig ne prend actuellement pas en charge les valeurs de ce type.

### PR DESCRIPTION
**Pull request by MattiSG**
Fixes #22.

### CODE REVIEW

File: migrations/v1.0_v1.1.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+stream.on('end', function() {
+    console.log('completed!');
+    process.exit();
+});
+
+stream.on('error', function(err) {
+    console.trace(err);
+});
+
```
Comment (fpagnoux) [inline migrations/v1.0_v1.1.js:52]: Some manual work will be needed after this migration, (some tests will be cleaned but will remain empty).

Comment (MattiSG) [inline migrations/v1.0_v1.1.js:52]: Good catch. On it.

**Comment by fpagnoux** [comment]
There is one red test remaining.
Otherwise, GTM for me.

**Comment by MattiSG** [comment]
> There is one red test remaining.

Which one? When I ran this on a copy of the production database it said it had nothing to do.

**Comment by fpagnoux** [comment]
``` sh
> ludwig-api@1.1.0 test /Users/fpagnoux/dev/mes-aides/ludwig-api
> TARGETS='lib/**/*.js fixtures/*.js *.js'; jshint $TARGETS && jscs $TARGETS

lib/models/acceptanceTest.js: line 75, col 46, Use '!==' to compare with '0'.

1 error
npm ERR! Test failed.  See above for more details.
```

**Comment by fpagnoux** [comment]
This is the test pointed out by travis

**Comment by MattiSG** [comment]
Haha ok there was an ambiguity with “test”, I thought it was a Ludwig test  ;)
Should be fixed now.

**Pull request merged**